### PR TITLE
feature: add some params for java-dependency-check action.

### PR DESCRIPTION
**Pull request by xautjzd**
## Description

Add some params for java-dependency-check action.

## Checklist

1. Specify dependency-check maven plugin goal.
2. Specify CVSS score.

### CODE REVIEW

File: actions/java-dependency-check/1.0/internal/pkg/conf/conf.go [hunk 1]
```diff
@@ -13,9 +13,11 @@ type Conf struct {
...
-	AutoUpdateNVD        bool   `env:"ACTION_AUTO_UPDATE_NVD" default:"false"`
-	MavenPluginVersion   string `env:"ACTION_MAVEN_PLUGIN_VERSION" default:"6.3.1"`
-	MavenSettingsXMLPath string `env:"ACTION_MAVEN_SETTINGS_XML_PATH" required:"false"`
+	CodeDir              string  `env:"ACTION_CODE_DIR" required:"true"`
+	Debug                bool    `env:"ACTION_DEBUG" default:"false"`
+	Goal                 string  `env:"ACTION_DEPENDENCY_CHECK_GOAL" default: "check"`
+	AutoUpdateNVD        bool    `env:"ACTION_AUTO_UPDATE_NVD" default:"false"`
+	MavenPluginVersion   string  `env:"ACTION_MAVEN_PLUGIN_VERSION" default:"6.3.1"`
+	MavenSettingsXMLPath string  `env:"ACTION_MAVEN_SETTINGS_XML_PATH" required:"false"`
+	FailBuildOnCVSS      float64 `env: "ACTION_FAIL_BUILD_ON_CVSS" default:"9.0"`
```
Comment (sfwn) [inline actions/java-dependency-check/1.0/internal/pkg/conf/conf.go:22]: remove blank at `env: "`

File: actions/java-dependency-check/1.0/spec.yml [hunk 2]
```diff
@@ -34,21 +37,28 @@ params:
...
+  - name: fail_build_on_cvss
+    type: float64
+    desc: ${{ i18n.params.fail_build_on_cvss.desc }}
 locale:
   zh-CN:
     desc: Java 依赖漏洞检测，并提供检测报告下载。
     displayName: Java 工程漏洞检测
     params.auto_update_nvd.desc: 自动更新 NVD 漏洞数据库；内置数据库已为最新，一般情况下无需打开该选项。该选项需要联网，网络不畅会导致更新失败。
     params.code_dir.desc: 代码目录，例如 ${git-checkout}
+    params.goal.desc: 用户指定漏洞检测插件的 goal, 可选值有 aggregate, check
```
Comment (sfwn) [inline actions/java-dependency-check/1.0/spec.yml:49]: add more description for aggregate and check

Comment (sfwn) [reply actions/java-dependency-check/1.0/spec.yml:49]: adjust README.md

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by xautjzd** [comment]
fixed

File: actions/java-dependency-check/1.0/spec.yml (hunk 2 above)
Comment (huangzx618) [reply actions/java-dependency-check/1.0/spec.yml:51]: do you mean this?
--用户指定漏洞检测报告的目标形式
* aggregate：创建汇总的依赖性检查报告
* check：在指定目标内创建dependency-check-report.html

**Review by sfwn — DISMISSED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**